### PR TITLE
Add lecto.ai

### PR DESCRIPTION
**Pull request by AgrimPrasad**
Machine Translation API provider with Free tier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -634,6 +634,7 @@ Table of Contents
 
   * [crowdin.com](https://crowdin.com/) — Unlimited projects, unlimited strings and collaborators for Open Source
   * [gitlocalize.com](https://gitlocalize.com) - Free and unlimited for both private and public repositories
+  * [Lecto](https://lecto.ai/) - Machine Translation API with Free tier (30 free requests, 1000 translated characters per request). Integrated with Loco Translate Wordpress plugin.
```
Comment (ripienaar) [inline README.md:637]: It seems more like a free trial that gives some credits rather than a rolling free tier.  That would not qualify for the list

Comment (AgrimPrasad) [reply README.md:637]: Thanks for the review. Would the Basic Free Tier here with monthly rolling free requests qualify? https://rapidapi.com/lecto-lecto-default/api/lecto-translation/pricing

Comment (ripienaar) [reply README.md:637]: Sure if that was on the https://lecto.ai/ site in a way that was obvious to find

Comment (AgrimPrasad) [reply README.md:637]: Sure, we've added prominent links to our RapidAPI and APILayer offerings here now: https://lecto.ai/#features

Both of these have rolling free tiers.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**